docs/samples/calling/app.js

```diff
@@ -610,6 +609,16 @@ function createCall(e) {
     document.getElementById('remote-audio').srcObject = new MediaStream([track]);
   });
 
+  localAudioStream.on('system-mute-state-change', (systemMuted) => {
```

Comment:
Are we not going to make any changes to the call object on receiving this?

Reply:
No changes are required. It works without the call object changes